From 4a5e8d63ef41ed549ec297c534fb5d5ef3cffdd2 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <B38613@freescale.com>
Date: Wed, 22 Aug 2012 11:35:24 +0800
Subject: [PATCH 046/281] ENGR00220371 Ethernet:sometimes got DHCP request
 error when up

sometimes got "E/Ethernet( 2359):DHCP request error:Timed out waiting
for dhcpcd to start" error. This is caused by ifc_ctl_sock return EPERM
 which means no permission to run on eth0. This will cause no ethernet
connected message sent out in framwork.

Signed-off-by: Jianzheng Zhou <B38613@freescale.com>
---
 imx5x/init.rc    | 2 +-
 imx6/etc/init.rc | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/imx5x/init.rc b/imx5x/init.rc
index c409b88..1182531 100644
--- a/imx5x/init.rc
+++ b/imx5x/init.rc
@@ -572,7 +572,7 @@ service dhcpcd_wlan0 /system/bin/dhcpcd -ABKL
     disabled
     oneshot
 
-service dhcpcd_eth0 /system/bin/dhcpcd -AL
+service dhcpcd_eth0 /system/bin/dhcpcd -ABKL
     class late_start
     disabled
     oneshot
diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index dbdf013..71874de 100644
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -586,7 +586,7 @@ service dhcpcd_wlan0 /system/bin/dhcpcd -ABKL
     disabled
     oneshot
 
-service dhcpcd_eth0 /system/bin/dhcpcd -AL
+service dhcpcd_eth0 /system/bin/dhcpcd -ABKL
     class late_start
     disabled
     oneshot
-- 
1.8.0

